modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/clients/APIPublisherRestClient.java

```diff
@@ -584,14 +557,176 @@ public HttpResponse changeAPILifeCycleStatusToPublish(APIIdentifier apiIdentifie
 	 * @param provider - Provider Name of the API
 	 * @param version  - Version of the API
 	 * @return HttpResponse -  Response of the getAPI request
-	 * @throws Exception - Exception Throws in checkAuthentication() and when do the REST service calls to get the
-	 *                   API information.
+	 * @throws APIManagerIntegrationTestException - Exception Throws in checkAuthentication() and when do the REST service calls to get the
+	 *                                            API information.
 	 */
 	public HttpResponse getApi(String apiName, String provider, String version)
-			throws Exception {
-		checkAuthentication();
-		return HttpRequestUtil.doPost(new URL(backEndUrl + "/publisher/site/blocks/listing/ajax/item-list.jag")
-				, "action=getAPI&name=" + apiName + "&version=" + version + "&provider=" + provider + "", requestHeaders);
+			throws APIManagerIntegrationTestException {
+		try {
+			checkAuthentication();
+			return HttpRequestUtil.doPost(new URL(backEndUrl + "/publisher/site/blocks/listing/ajax/item-list.jag")
+					, "action=getAPI&name=" + apiName + "&version=" + version + "&provider=" + provider + "", requestHeaders);
+		} catch (Exception e) {
+			throw new APIManagerIntegrationTestException("Exception when retrieving a API", e);
+
+		}
+
+	}
+
+	/**
+	 * Retrieve the Tier Permission Page
+	 *
+	 * @return HttpResponse - Response that contains the Tier Permission Page
+	 * @throws APIManagerIntegrationTestException - Exception throws from checkAuthentication() method and
+	 *                                            HttpRequestUtil.doGet() method call
+	 */
+	public HttpResponse getTierPermissionsPage() throws APIManagerIntegrationTestException {
+
+		try {
+			checkAuthentication();
+			return HttpRequestUtil.doGet(backEndUrl + "/publisher/site/pages/tiers.jag", requestHeaders);
+		} catch (Exception e) {
```

Comment:
This generic exception is thrown from test framework. Which need to be fix in next release.